src/fragments/lib-v1/auth/js/password_management/reset_password.mdx

```diff
@@ -0,0 +1,61 @@
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { resetPassword, ResetPasswordOutput } from 'aws-amplify/auth';
+
+async function handleResetPassword(username: string) {
+	try {
+		const output = await resetPassword({ username });
```

Comment:
Should this call the util function? Also, I wonder if we want to call these utils something like `handleResetPasswordNextSteps` or something 🤔 Open to suggestions

```suggestion
		const output = await resetPassword({ username });
		
		handleResetPasswordOutput(output);
```

Reply:
yeah the naming make sense to me. It's clear that a nextStep should be handled.